core/jobs/batch_jobs/contributor_admin_stats_jobs.py

```diff
@@ -558,62 +594,108 @@ def transform_question_contribution_stats(
 
         entity_id = contributor_user_id
 
+        # Collects all the debug logs.
+        debug_logs: str = (
+            'Question submitter ID: %s.\n' % contributor_user_id)
+
+        by_topic_id = lambda m: m.topic_id
+
+        skill_ids_with_question_suggestions = sorted({
+            v.target_id for v in general_suggestion_stats})
+        debug_logs += (
+            'Unique skill IDs with question suggestion: \n')
```

Comment:
I think it would be better if we can print these logs only for users who cause the failure. Ohterwise it would be difficult to debug?